torch/csrc/Exceptions.cpp

```diff
@@ -60,8 +60,8 @@ could not be completed because the input matrix is singular.",
   // NOLINTNEXTLINE(bugprone-assignment-in-if-condition)
   ASSERT_TRUE(
       THPException_OutOfMemoryError = PyErr_NewExceptionWithDoc(
-          "torch.cuda.OutOfMemoryError",
-          "Exception raised when CUDA is out of memory",
+          "torch.OutOfMemoryError",
+          "Exception raised when device is out of memory",
           PyExc_RuntimeError,
```

Comment:
Another method is to rename `torch._C._OutOfMemoryError` to `torch.OutOfMemoryError` as below:
```cpp
  PyTypeObject* type = (PyTypeObject*)THPException_OutOfMemoryError;
  type->tp_name = "torch.OutOfMemoryError";

  ASSERT_TRUE(
      PyModule_AddObject(
          module, "OutOfMemoryError", THPException_OutOfMemoryError) == 0);
```
Then you can remove `OutOfMemoryError`'s description from the `torch.rst`. Meanwhile, change `_OutOfMemoryError` to `OutOfMemoryError` in `torch/_C/__init__.pyi.in` and `torch/cuda/\_\_init\_\_.py`
I think this is better.

Reply:
ok，updated.

Reply:
Thanks for your update, please fix the CI failure.

Reply:
> Thanks for your update, please fix the CI failure.

Done. All tests have passed now.